lists-service/cicd/build/pipeline/build_lists_buildspec.yaml

```diff
@@ -0,0 +1,48 @@
+version: 0.2
+
+env:
+  shell: bash
+  variables:
+    JAVA_TOOL_OPTIONS: -Dhttps.protocols=TLSv1.2
+
+phases:
+  install:
+    commands:
+      - echo Installing dependencies...
+      - apt update -y
+        && cat /etc/lsb-release
+        && apt-get -q -y install openjdk-11-jdk
+        && apt-get -q -y install unzip
+        && apt-get -q -y install zip
+        && apt-get -q -y install curl
+        && curl -s "https://get.sdkman.io" | bash
+        && chmod a+x /root/.sdkman/bin/sdkman-init.sh
+        && source /root/.sdkman/bin/sdkman-init.sh
+        && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+        && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+        && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        && apt-get update
+        && apt-get install -y docker-ce
+        && apt-get install -y maven
```

Comment:
A lot of these packages are already present in the standard build containers or not needed. [Userdetails ended up needing only a single package](https://github.com/AtlasOfLivingAustralia/userdetails/blob/feature/containerization/cicd/package/pipeline/build_userdetails_buildspec.yaml#L13). It will save build time and simplify things if we can cut this down to only whats needed